contrib/minimal.lua

```diff
@@ -0,0 +1,30 @@
+vim.cmd [[set runtimepath=$VIMRUNTIME]]
```

Comment:
Don't know how to review an entire file so picked the first line.

Don't we also need a template for people who use init.vim? If not then maybe we should mention "If you use init.lua use the minimal.lua template, otherwise just minimize your current config." .

Reply:
I think there's only a tiny subset of issues that _require_ an `init.vim` (as in, cannot be reproduced  with an `init.lua`). And vimscript config can be used in the config block by wrapping it in `vim.cmd[[ ... ]]`.

Could be explained more, of course, but again, maybe not here?

Reply:
You may resolve this conversation if you'd like. I don't have the permission to do it myself.

Reply:
Same goes for the other conversations really.